index.html

```diff
@@ -18,189 +18,216 @@
 		<header class="navbar">
 			<a href="#" class="logo">Fernando</a>
 			<nav>
-				<i class="fas fa-bars"></i>
+				<ul>
+					<li><a href="#">Portfolio</a></li>
+					<li><a href="#">About</a></li>
+					<li><a href="#">Contact</a></li>
+				</ul>
```

Comment:
- [ ] The desktop navigation fails to transform to mobile navigation when the screen size reduces. It would be nice if you added it too because they are both meant to appear though at different screen sizes.